### ISSUE TITLE
Plugin can not distingiush between external and internal modules

### ISSUE THREAD

**Issue by miraage** [benmosher/eslint-plugin-import#1579]
Repo: https://github.com/miraage/eslint-import-plugin-bug
ESLint: https://github.com/miraage/eslint-import-plugin-bug/blob/master/.eslintrc.js
File: https://github.com/miraage/eslint-import-plugin-bug/blob/master/src/App.js

https://github.com/miraage/eslint-import-plugin-bug/blob/master/src/App.js#L1-L3

```
/Users/miraage/Projects/eslint-import-plugin-bug/src/App.js
  1:1  warning  There should be no empty line within import group  import/order
```

I believe it should not report a warning.

(edit: removed wrong assumtions)

**Comment by miraage** [comment]
I think I've figured it. I'm using eslint-import-resolver-typescript, which has bug.

**Comment by ryanheathers** [comment]
@miraage what did you figure out for this? I don't see any bug reports over at the `eslint-import-resolver-typescript` plugin. 

My issue sounds similar to yours. I'm getting complaints about when using external imports and Typescript aliases, with no obvious solution

e.g.
```
import React from 'react' // external import

import { myFunc} from '@src/myInternalModule // internal import
```
Yet this causes the `There should be no empty line within import group  import/order' error` even though these **ought** to be separate groups in the eyes on the import/order rule.

Here is my config inside `.eslintrc`
```
'import/order': [
  'error',
  {
  'newlines-between': 'always',
  groups: [
    ['builtin', 'external'],
     'internal',
    ['sibling', 'parent', 'index'],
  ],
 },
],
```
